Lib/pdb.py

```diff
@@ -164,8 +167,8 @@ class _ModuleTarget(str):
     def check(self):
         try:
             self._details
-        except Exception:
-            traceback.print_exc()
+        except Exception as e:
+            print(f"{type(e).__name__}: {e}")
```

Comment:
Can you make it something like
```py
        try:
            self._details
        except ... : # Something you expect
            print(f"{type(e).__name__}: {e}")
        except Exception:
            traceback.print_exc()
```

to leave the general case untouched.

Reply:
Looking at the `runpy._get_module_details` source, `ImportError` seems to be the only type of exception that can be raised here.